pkg/controller/util/endpoint/controller_utils_test.go

```diff
@@ -830,7 +830,7 @@ func TestEndpointsEqualBeyondHash(t *testing.T) {
 			expected: false,
 		},
 		{
-			name: "Pod resourceVersion changed",
```

Comment:
shouldn't we keep this test case as-is, and add a new test case to exercise "removed"?

Reply:
done, I removed it because I thought there is no longer Pod resourceVersion changed case in main code. But yes, the function is still supposed to return true for this case.